mtp_bank_admin/apps/bank_admin/utils.py

```diff
@@ -67,3 +62,27 @@ def escape_csv_formula(value):
     if isinstance(value, str) and value.startswith('='):
         return "'" + value
     return value
+
+
+class WorkdayChecker:
+
+    def __init__(self):
+        response = requests.get('https://www.gov.uk/bank-holidays.json')
+        if response.status_code == 200:
+            self.holidays = [
+                datetime.strptime(holiday['date'], '%Y-%m-%d').date() for holiday in
+                response.json()['england-and-wales']['events']
+            ]
+        else:
+            raise RuntimeError(
```

Comment:
wouldn't this cause a 500? i wonder if a fallback or a message to the user would be better…